setup.md

```diff
@@ -0,0 +1,334 @@
+need to return to this folder enter the following command:
+
+```
+cd %HOMEPATH%
+```
+
+**_Seeing and navigating the contents of folders_**
+
+One of the main uses of the terminal is to allow users to quickly see and navigate
+through the various files and folders on your computer.
+
+You can see what files are in the current folder you are accessing using:
+
+```
+dir
+```
+
+You can change from one folder to another (in this case named 'newfolder') by doing:
+
+```
+cd newfolder
+```
+
+If you want to know where you are amongst all your folders you can do:
+
+```
+cd
+```
+
+This will give you the full path to the current folder.
+
+If you want to delete (remove) a file do:
+
+```
+del file
+```
+
+If you want to rename a file do:
+
+```
+rename file newfilename
+```
+
+If you want to print the contents of a file do:
+
+```
+more file
+```
+
+### Using editors
+
+At several points during the tutorials you might be asked to edit a file.
+
+There are several editors available and generally we recommend you use
+the one that you are most accustomed with (if possible).
+
+Here are our recommendations for editors that can be invoked directly from
+the command line:
+
+#### linux/macOS
+
+- [gedit](https://help.gnome.org/users/gedit/stable/) (recommended for linux, usually pre-installed)
+- [nano](https://wiki.gentoo.org/wiki/Nano/Basics_Guide) (recommended for linux & macOS, pre-installed)
+- [TextEdit](https://support.apple.com/en-gb/guide/textedit/welcome/mac) (for macOS, pre-installed.  From the terminal prompt you can issue `open -a TextEdit`)
+- [vim](https://www.vim.org/download.php)
+
+#### Windows
+
+- notepad (recommended)
+
+Built into Windows, just type `notepad filename` in your terminal. This will open a
+window where you can edit the file, save and then close the window to get back to
+your terminal.
+
+- [notepad++](https://notepad-plus-plus.org/)
+- [vim](https://www.vim.org/download.php)
+
+
+## 3. Downloading a local copy of the course materials
+
+In order to carry out the exercies presented in the tutorials you will need to
+download a local copy of the course materials. There are two ways of doing this;
+
+### Recommended: download a copy from github
+
+Simply go to the tutorial github page: https://github.com/bigginlab/OxCompBio
+
+On the top left-hand side of the page you will see a green button titled "Code".
+
+Click on it and then click on "Download ZIP".
+
+![download zip](./images/main-setup/oxcompbio-zipdownload.png)
+
+This will download a copy of the course materials to your computer. Then simply
+move the zip file to some convenient folder (ideally easily accessible through your
+terminal), and extract it (macOS: double click, windows: right-click and extract all).
+
+This will create a folder named `OxCompBio`.
+
+### Not recommended: Git clone 
+This is not a recommended option, but can be useful if you are using linux/macOS
+and are already quite familiar with `git`.
```

Comment:
I'm not sure if `git` is installed by default

Reply:
🤷‍♂️ if they don't have git then they shouldn't be using this method 🙃 I did make it clear it's not the recommended option.

Reply:
Should we just remove it? If they know how to use `git` they will likely know how to clone the repository anyway.

Reply:
Sure, I don't have any particular attachment to it.

Can we confirm that downloading the zip will lead to `OxCompBio-master` as the top level directory? I'll need to change the python instructions accordingly.

Reply:
Done now, I'll try to refer to `OxCompBio/OxCompBio-master` where possible.